editor/import/scene_import_settings.h

```diff
@@ -185,7 +214,20 @@ class SceneImportSettingsDialog : public ConfirmationDialog {
 
 	String base_path;
 
-	MenuButton *action_menu = nullptr;
+	enum Tool {
+		TOOL_SELECT,
+		TOOL_SELECT_LIST,
+		TOOL_SELECT_OBJECT,
+		TOOL_SELECT_MESH,
+		TOOL_SELECT_MATERIAL,
+		TOOL_PREVIEW_MATERIALS,
+		TOOL_MAX
+	};
+
+	Button *tool_buttons[TOOL_MAX];
```

Comment:
```suggestion
	Button *tool_buttons[TOOL_MAX] = {};
```
Think this is a good way to initialize these cleanly